versions/3.0.md

```diff
@@ -3368,7 +3368,7 @@ animals:
 #### <a name="securitySchemeObject"></a>Security Scheme Object
 
 Allows the definition of a security scheme that can be used by the operations.
-Supported schemes are HTTP authentication, an API key (either as a header or as a query parameter) and OAuth2's common flows (implicit, password, application and access code).
+Supported schemes are HTTP authentication, an API key (either as a header or as a query parameter), OAuth2's common flows (implicit, password, application and access code) as defined in [RFC6749](https://tools.ietf.org/html/rfc6749), and [OpenID Connect](http://openid.net/specs/openid-connect-discovery-1_0.html).
```

Comment:
Did you mean `http://openid.net/specs/openid-connect-core-1_0.html` not `http://openid.net/specs/openid-connect-discovery-1_0.html`?